tools/install_runner_prerequisites.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+set -eu
+
+MARKER=/etc/opt/oxide/NO_INSTALL
+if [[ -f "$MARKER" ]]; then
+  echo "This system has the marker file $MARKER, aborting." >&2
+  exit 1
+fi
+
+# Set the CWD to Omicron's source.
+SOURCE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+cd "${SOURCE_DIR}/.."
+
+function on_exit
+{
+  echo "Something went wrong, but this script is idempotent - If you can fix the issue, try re-running"
+}
+
+trap on_exit ERR
+
+# Parse command line options:
+#
+# -y  Assume "yes" intead of showing confirmation prompts.
+ASSUME_YES="false"
+while getopts yp flag
```

Comment:
```suggestion
while getopts y flag
```

Reply:
I actually went the other way - I made both calls to `getopts` take the same arguments. See comment below.